plugins/windows_event.yaml

```diff
@@ -87,3 +103,20 @@ pipeline:
     start_at: {{ $start_at }}
     output: {{ .output }}
 # {{ end }}
+
+# {{$output := .output}}
```

Comment:
I don't really understand why we are setting these here? is there a reason fro this?

Reply:
Because of the scope of the . when the array is iterated over the {{.id}} and {{.output}} had to be stored in a variable to resolve the scope issue.

Reply:
Ahh this was what you were talking about, okay then! 👍